src/hotspot/share/runtime/mutexLocker.cpp

```diff
@@ -306,7 +306,7 @@ void mutex_init() {
 #endif
   MUTEX_DEFN(DumpTimeTable_lock              , PaddedMutex  , nosafepoint);
   MUTEX_DEFN(CDSLambda_lock                  , PaddedMutex  , nosafepoint);
-  MUTEX_DEFN(DumpRegion_lock                 , PaddedMutex  , nosafepoint);
+  MUTEX_DEFN(DumpRegion_lock                 , PaddedMutex  , nosafepoint-1); // Holds DumpTimeTable_lock
```

Comment:
There's a MUTEX_DEFL where you can define the mutex in terms of the locks that it takes out.